ci/run.bat

```diff
@@ -0,0 +1,4 @@
+dub test --skip-registry=all --compiler=%DC%
+::rdmd --compiler=%DC% ./tests/runner.d --compiler=%DC% -cov
+dub build --skip-registry=all --compiler=%DC%
+dub build -c client --skip-registry=all --compiler=%DC%
```

Comment:
Why does this need to be a `bat` script ?

Reply:
vcvars64 environment variables are readily available.
run without writing `bat` script Should it be implemented in `run.sh`?

Reply:
run in bash while maintaining msbuild environment variable.Could not find a way to run sh.
Currently, this method is maintained. Let me know if there's a better way later.